documents/privacy.md

```diff
@@ -0,0 +1,301 @@
+-   the servers that encode hashes for Safe Places are approximately 10x more powerful
+-   the servers used by an attacker are approx 100x as powerful as the smartphone.
+-   servers of such power could be rented in the cloud (e.g. AWS) for 10c/hour
+
+(we are actively working to validate these assumptions)
+
+| **Activity**                                             | **Smartphone CPU seconds (SCS)** | **Estimated CPU time**         |
+| -------------------------------------------------------- | -------------------------------- | ------------------------------ |
+| Smartphone hashes 1 location                             | 4.6                              | 4.6 seconds                    |
+| Smartphone CPU usage/day                                 | 1.3k                             | 22 mins                        |
+| Safe Places publish (100 cases)                          | 100k                             | 3 hours                        |
+| Safe Places CPU usage/day                                | 200k                             | 6 hours                        |
+| Attack on single geohash, 1 day                          | 288                              | 3 seconds                      |
+| Attack on 1 sq km, 1 day                                 | 400k                             | 1 hour 6 mins (est. cost 12 c) |
+| Attack on 10km x 10km, 14 days                           | 560M                             | 2 months (est. cost $144.00)   |
+| Attack on 3000 sq km, 14 days (e.g. a typical US County) | 16.8B                            | 5 years (est. cost $4,560)     |
+
+
+
+Relating this to our privacy goals articulated above
+
+-   There is no significant risk to undiagnosed users (whose data is never shared), or diagnosed users (since the data they have
+    provided, even if it were decrypted, is anonymized and redacted).
+    
+-   There is, unfortunately, a vulnerability for venues where diagnosed users have been - particularly in the case where an attacker has in mind a small area and a narrow time window, which they wish to evaluate for points of concern.
+
+However, while such attacks are possible, it nevertheless will take significant technical skill and effort to conduct such an attack, and the rewards are fairly minimal. Our expectation is that this will lead to few such attacks will be undertaken, and that this will be a negligible channel for disclosure of information about which venues were attended by people later diagnoses with COVID.
+
+In practice, we expect that the level of information leaked by this channel will be negligible, as compared to the level of information leaked through informal networks of local gossip.
+
+Some Clarifying Points on Efficacy
+----------------------------------
+
+The main purpose of this article is to present our approach to data privacy. However some of the points raised here may raised broader concerns around efficacy of location-based exposure notifications.
```

Comment:
However some of the points raised here may **raised** broader concerns should be However some of the points raised here may **raise** broader concerns